### PR TITLE
feat: stress schedule jobs

### PR DESCRIPTION
**Pull request by pygman**

### CODE REVIEW

File: offchain-modules/packages/x/src/ckb/tx-helper/generator.ts [hunk 1]
```diff
@@ -362,6 +362,11 @@ export class CkbTxGenerator extends CkbTxHelper {
     const sudtCells = await this.collector.collectSudtByAmount(searchKey, amount);
     const total = sudtCells.map((cell) => utils.readBigUInt128LE(cell.data)).reduce((a, b) => a + b, 0n);
     if (total < amount) {
+      logger.error(
```
Comment (mkxbl) [inline offchain-modules/packages/x/src/ckb/tx-helper/generator.ts:365]:
```suggestion
      logger.warn(
```

File: offchain-modules/packages/x/src/ckb/tx-helper/generator.ts [hunk 2]
```diff
@@ -747,6 +752,11 @@ export class CkbTxGenerator extends CkbTxHelper {
...
```
Comment (mkxbl) [inline offchain-modules/packages/x/src/ckb/tx-helper/generator.ts:755]:
```suggestion
      logger.warn(
```

File: offchain-modules/packages/scripts/src/utils/eth_batch_test.ts [hunk 3]
```diff
@@ -162,9 +166,13 @@ export async function lock(
...
-    const lockTxHash = (await provider.sendTransaction(signedLockTxs[i])).hash;
-    await asyncSleep(intervalMs);
-    lockTxHashes.push(lockTxHash);
+    try {
+      const lockTxHash = (await provider.sendTransaction(signedLockTxs[i])).hash;
+      await asyncSleep(intervalMs);
+      lockTxHashes.push(lockTxHash);
+    } catch (e) {
+      logger.error(e.stack);
+    }
```
Comment (mkxbl) [inline offchain-modules/packages/scripts/src/utils/eth_batch_test.ts:175]: should throw e in catch block ?

Comment (pygman) [reply offchain-modules/packages/scripts/src/utils/eth_batch_test.ts]: added throw e, and stop running, troubleshoot errors manually

File: offchain-modules/packages/x/src/ckb/tx-helper/generator.ts (hunk 2 above)
Comment (pygman) [reply offchain-modules/packages/x/src/ckb/tx-helper/generator.ts]: fixed

File: offchain-modules/packages/x/src/ckb/tx-helper/generator.ts (hunk 1 above)
Comment (pygman) [reply offchain-modules/packages/x/src/ckb/tx-helper/generator.ts]: fixed

**Pull request merged**